.github/workflows/integration-tests.yml

```diff
@@ -118,7 +118,8 @@ jobs:
           TRITON_INTERPRET: "1"
         run: |
           cd python/test/unit
-          python3 -m pytest -vvv -n 4 -m interpreter language/test_core.py --device cpu
+          python3 -m pytest -vvv -n 8 -m interpreter language/test_core.py --device cpu
+          python3 -m pytest -vvv -m interpreter language/test_standard.py --device cpu
```

Comment:
Add a comment explaining why test_standard cannot be run with -n 8?

Reply:
Yeah, that's clever. Thanks